app/src/main/java/org/cru/godtools/fragment/ToolDetailsFragment.java

```diff
@@ -275,29 +281,13 @@ private void updatePinShortcutAction() {
     }
 
     private void updateViews() {
+        setUpViewPager();
```

Comment:
you are still setting up the view pager every time the view is being updated

Reply:
this should only be called when the view is created, not every time `updateViews()` is called

Reply:
@frett I was unable to remove the setAdapter from the `updateViews`.  I tried doing a notifyDataSet Change and InitiateItem but I wasn't able to see the text in the view pager.